source/process/casestudy.rst

```diff
@@ -14,8 +14,12 @@ The process is approximately as follows:
 1. Customer has completed a deployment of Mattermost and is open to a case study. 
 2. Customer case study contact is introduced to a Mattermost case study coordinator over email and completes a questionnaire (below). 
 3. Mattermost case study coordinator arranges a 30-minute phone or video call with the customer for a discussion around the questionnaire and deployment. 
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval.
+5. Mattermost case study coordinator politely checks in with the customer every 2 business days for an estimate of the draft case study review time.
+6. The case study coordinator also commissions a banner with a clear heading and a nice background image to be included in the case study blog post.
+7. The case study coordinator creates a draft blog post (unpublished) on the Mattermost staging server. The draft will then be reviewed for formatting, grammar, and so on.
```

Comment:
Does this step happen after we get the post from the customer?

Other minor notes:
1. Is `(unpublished)` needed?
2. Is `Mattermost staging server` needed? (if this page is for customers, probably not)
3. Can we link to style guidelines? E.g. what formatting should it follow? Perhaps the guidelines for blog posts?